app/src/main/res/layout/fragment_sell_payload.xml

```diff
@@ -0,0 +1,103 @@
+        app:navigationIcon="@drawable/ic_toolbar_back"
+        app:title="@string/sell_payload_title" />
+
+    <TextView
+        android:id="@+id/textViewSell"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
+        android:text="@string/common_cash_out"
+        app:layout_constraintBottom_toTopOf="@id/editTextToken"
+        app:layout_constraintStart_toStartOf="@id/editTextToken" />
+
+    <TextView
+        android:id="@+id/textViewSellAmount"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="4dp"
+        android:textAppearance="@style/UiKit.TextAppearance.Regular.Label1"
+        android:textColor="@color/text_sky"
+        app:layout_constraintBottom_toBottomOf="@id/textViewSell"
+        app:layout_constraintStart_toEndOf="@id/textViewSell"
+        app:layout_constraintTop_toTopOf="@id/textViewSell"
+        tools:text="all 4.53 SOL" />
+
+
+    <org.p2p.uikit.components.UiKitEditText
+        android:id="@+id/editTextToken"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginTop="120dp"
+        app:layout_constraintTop_toBottomOf="@id/toolbar" />
+
+    <org.p2p.uikit.components.UiKitEditText
+        android:id="@+id/editTextAmount"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginTop="8dp"
+        android:textAppearance="@style/UiKit.TextAppearance.SemiBold.Title1"
+        app:layout_constraintTop_toBottomOf="@id/editTextToken" />
+
+    <TextView
+        android:id="@+id/textViewRate"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginTop="20dp"
+        android:background="@drawable/bg_app_top_rounded"
+        android:padding="16dp"
+        android:textAppearance="@style/UiKit.TextAppearance.Regular.Label1"
+        android:textColor="@color/text_mountain"
+        app:layout_constraintTop_toBottomOf="@id/editTextAmount"
+        tools:text="1 SOL ≈ 34.65 EUR" />
+
+    <TextView
+        android:id="@+id/textViewFee"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginTop="1dp"
+        android:background="@drawable/bg_app_bottom_rounded"
+        android:padding="16dp"
+        android:textAppearance="@style/UiKit.TextAppearance.Regular.Label1"
+        android:textColor="@color/text_mountain"
+        app:layout_constraintTop_toBottomOf="@id/textViewRate"
+        tools:text="Included fee 0.6 SOL" />
+
+    <org.p2p.uikit.components.UiKitButton
+        android:id="@+id/buttonSend"
```

Comment:
buttonCashOut?

Reply:
it was send, when I created it. After they renamed the button